examples/demo_app/package.json

```diff
@@ -1,20 +1,20 @@
 {
-  "name": "demo_app",
+  "name": "demo-app",
```

Comment:
This change comes from https://github.com/rails/rails/commit/e22bcc24094d87a41385a0f6053cecb1b8807a86